apm-sniffer/apm-sdk-plugin/grpc-1.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/grpc/v1/AsyncUnaryRequestCallCallInterceptor.java

```diff
@@ -36,7 +36,7 @@
 public class AsyncUnaryRequestCallCallInterceptor implements StaticMethodsAroundInterceptor {
     @Override public void beforeMethod(Class clazz, Method method, Object[] allArguments, Class<?>[] parameterTypes,
         MethodInterceptResult result) {
-        BlockingCallClientInterceptor originClientCall = (BlockingCallClientInterceptor)allArguments[0];
+        CallClientInterceptor originClientCall = (CallClientInterceptor)allArguments[0];
```

Comment:
Is this the key of the fix? Because of the incompatible type cast?

Reply:
Yes

Reply:
@ascrutae Please review as you are the author.